docs/source/customizations/coprocessor.mdx

```diff
@@ -595,6 +595,21 @@ If your coprocessor [returns a _different_ value](#responding-to-coprocessor-req
 <tr>
 <td>
 
+##### `has_next`
+
+`bool`
+
+</td>
+<td>
+
+When `stage` is `SupergraphRequest`, if present and `true` then there may be subsequent `SupergraphRequest` calls to the co-processor for each multi-part (@defer/subscriptions) response.
+
+</td>
```

Comment:
I think this should be `SupergraphResponse`.

Reply:
7caf6be